atlasdb-client/src/main/java/com/palantir/atlasdb/util/SuppressedException.java

```diff
@@ -19,14 +19,14 @@
     static final long serialVersionUID = 1L;
     private static final Class[] namedThrowables = {Error.class, RuntimeException.class, Exception.class};
 
-    SuppressedException(String message, Throwable cause) {
-        super(message, cause);
+    SuppressedException(String message) {
+        super(message);
     }
 
     public static Throwable from(Throwable throwable) {
         String message = String.format("%s [%s] occurred while processing thread (%s)",
                 highLevelType(throwable), throwable, Thread.currentThread().getName());
-        return new SuppressedException(message, throwable);
+        return new SuppressedException(message);
```

Comment:
is this actually what we want? it means we will lose stacktrace information for the suppressed exception, right?